router/router.go

```diff
@@ -118,6 +115,7 @@ func loadConfig() {
 	retryTimeWindow = config.GetDuration("Router.retryTimeWindowInMins", time.Duration(180)) * time.Minute
 	minRetryBackoff = config.GetDuration("Router.minRetryBackoffInS", time.Duration(10)) * time.Second
 	maxRetryBackoff = config.GetDuration("Router.maxRetryBackoffInS", time.Duration(300)) * time.Second
+	workerRequestBatchTimeout = config.GetDuration("Router.workerRequestBatchTimeoutInS", 5) * time.Second
```

Comment:
Let's start supporting multiple units of duration going forward? We can have fixed suffix list like 
"20s" for 20 seconds
"20ms" for 20 milli seconds etc.

Because we faced difficulty recently when we wanted tune a timeout configuration to milli seconds but it only supported seconds.